uPortal-webapp/src/main/webapp/scripts/portal-analytics.js

```diff
@@ -42,9 +42,16 @@
             return;
         }
 
+        const tab = document.querySelector('li.fl-tabs-active span');
```

Comment:
Really like seeing `const` here, I vastly prefer it over `var`.
Just to double check, what is the current browser support matrix?
Every evergreen browser does support `const` and has for a while, IE still does not.

Reply:
We no longer support IE, only Edge. 👏🏽